lib/src/lints/flutter/prefer_dedicated_media_query_method.dart

```diff
@@ -54,6 +54,38 @@ class PreferDedicatedMediaQueryMethod extends DartLintRule {
   ) {
     if (!context.pubspec.isFlutterProject) return;
 
+    context.registry.addPrefixedIdentifier((node) {
+      final targetType = node.prefix.staticType;
+
+      const mediaQueryDataChecker = TypeChecker.fromName(
+        'MediaQueryData',
+        packageName: 'flutter',
+      );
```

Comment:
We should probably move this to the type checker file

Reply:
Done